basestar-storage/src/test/java/io/basestar/storage/TestStorage.java

```diff
@@ -242,6 +247,32 @@ protected void testIndexes() throws IOException {
         });
     }
 
+    @Test
+    protected void testWithoutIndexes() throws IOException {
+
+        assumeTrue(supportsMissingIndexes());
+
+        final Storage storage = storage(namespace);
+
+        final ObjectSchema schema = namespace.requireObjectSchema(ADDRESS);
+
+        bulkLoad(storage, loadAddresses());
+
+        final Expression expr = Expression.parse("zip == '95334 CEDEX'");
+        final Page<Map<String, Object>> results = storage.query(Consistency.ATOMIC, schema, Immutable.map(), expr, ImmutableList.of(), Collections.emptySet()).page(EnumSet.of(Page.Stat.TOTAL), null, 100000).join();
+
+        final Set<String> ids = Sets.newHashSet("192");
+        assertEquals(ids, Sets.newHashSet(results.map(Instance::getId).getItems()));
+        if (results.getStats() != null && results.getStats().getTotal() != null) {
+            assertEquals(1, results.getStats().getTotal());
+        }
+        results.forEach(result -> {
+            // Ensure results are re-projected if necessary
+            assertNotNull(result.get("name"));
+            assertNotNull(result.get("company"));
+        });
```

Comment:
should the test have coverage for correctly getting next page? Since I saw new code for encoding/decoding paging info.

Reply:
added